py-polars/polars/series/datetime.py

```diff
@@ -1203,9 +1237,20 @@ def minutes(self) -> Series:
 
         """
 
+    @deprecate_renamed_function("total_seconds", version="0.19.3")
     def seconds(self) -> Series:
         """
-        Extract the seconds from a Duration type.
+        Extract the total seconds from a Duration type.
+
+        .. deprecated:: 0.19.3
+            Use :meth:`total_seconds` instead.
```

Comment:
I'm going to update the deprecation utils to automatically add these "deprecated" admonitions. For now it needs to be manual though.